### PR TITLE
card component

### PR DESCRIPTION
**Pull request by mghazidev**

### CODE REVIEW

File: public/index.html
Comment (mghazidev) [inline public/index.html:1]: don't use head Tag it is not a
 good practice

File: src/App.js
Comment (mghazidev) [inline src/App.js:1]: as we are using tailwind so dont write className="App":

File: src/Components/Timezone.jsx
Comment (mghazidev) [inline src/Components/Timezone.jsx:1]: good for now but we will create hooks for time functionality.

File: public/index.html
Comment (RabiaHameedKhan) [reply public/index.html:1]: where did i used the head tags?

Comment (mghazidev) [reply public/index.html:1]: dont use it it is not imp , for icons you can use aby library online.

**Pull request merged**